### PR TITLE
New download page on druid.io

### PR DESCRIPTION
**Pull request by guobingkun**
Include download links for mysql-metadata extesnion and tranquility.

Todo: Upload the new tarballs

Screenshot below
image

### CODE REVIEW

**Comment by fjy** [comment]
Can we include a link to installing the MySQL extension?

**Comment by guobingkun** [comment]
@fjy Done adding a link to installing MySQL extension.

**Comment by fjy** [comment]
:+1:

File: downloads.html [hunk 1]
```diff
@@ -24,12 +24,23 @@
...
+      <p>The current release candidate is tagged at version {{site.druid_rc_version}}.</p>
+
+      <p>Druid: <a href="http://static.druid.io/artifacts/releases/druid-{{ site.druid_rc_version }}-bin.tar.gz" onclick="trackDownload('link', 'http://static.druid.io/artifacts/releases/druid-{{ site.druid_rc_version }}-bin.tar.gz');return false;">druid-{{site.druid_rc_version}}-bin.tar.gz</a>.</p>
+
+      <p>MySQL Extension: <a href="http://static.druid.io/artifacts/releases/mysql-metadata-storage-{{ site.druid_rc_version }}.tar.gz" onclick="trackDownload('link', 'http://static.druid.io/artifacts/releases/mysql-metadata-storage-{{ site.druid_rc_version }}.tar.gz');return false;">mysql-metadata-storage-{{site.druid_rc_version}}.tar.gz</a>. (Due to licensing, we've separated MySQL extension from main Druid release. If you would like to use MySQL extension, please untar this tarball and follow the steps in <a href="/docs/{{site.druid_rc_version}}/operations/including-extensions.html">Include Extensions</a>)</p>
```
Comment (fjy) [inline downloads.html]: can we call this MYSQL metadata store extension?

Comment (guobingkun) [inline downloads.html]: @fjy done.

**Comment by guobingkun** [comment]
tarballs uploaded. Updated the new screenshot. Good to merge.

**Pull request merged**